hash/c64_cass.xml

```diff
@@ -2226,15 +2112,21 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 	</software>
 
 	<software name="bdash4h">
-		<description>Boulder Dash 4</description>
+		<description>Boulder Dash IV</description>
```

Comment:
This is also known as Boulder Dash: Construction Kit

Reply:
Yeah, the packaging's a bit ambiguous here.  The front says "Boulderdash IV: The Game; complete with Construction Kit" whereas the spine says "Boulderdash IV: Construction Kit".  I think they were trying to reassure people it was both a full game and a construction kit, not just one or the other.

Reply:
Suggest using the alt_title field in similar manner as below example:
	<software name="badcat">
		<description>Bad Cat</description>
		<year>1987</year>
		<publisher>Go!</publisher>
		<info name="alt_title" value="Street Cat"/>
I have been doing this whilst I have been going along adding the C64 Tape Archive V2.

Reply:
Thanks again!  I've added the main titles from the front and spine of the cover artwork: [e015bad](https://github.com/mamedev/mame/pull/9307/commits/e015badd1418afc4fb47b938e2bd25893c06c45a)